tests/integration/targets/vm/tasks/02_shutdown_cases.yml

```diff
@@ -36,7 +36,7 @@
       - mytag2
     memory: "{{ '512 MB' | human_to_bytes }}"
     vcpu: 2
-    attach_guest_tools_iso: true
+    attach_guest_tools_iso: false
```

Comment:
`attach_guest_tools_iso` wasn't working correctly at the time when this test was written.
I changed the value to `false` everywhere where this was the case to keep the test integrity.
Made a separate test in the vm_git folder to test `attach_guest_tools_iso`.